websites/D/Daily Mail/metadata.json

```diff
@@ -0,0 +1,23 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.10",
+	"author": {
+		"id": "358402930191106049",
+		"name": "ion606"
+	},
+	"service": "Daily Mail",
+	"description": {
+		"en": "The official website for the Daily Mail publication"
+	},
+	"url": "www.dailymail.co.uk",
+	"version": "1.0.0",
```

Comment:
There is a new field called `matches`, which is required for new Presences.
These are an array of strings which need to follow https://developer.chrome.com/docs/extensions/develop/concepts/match-patterns.